frontend/server/src/DAO/Identities.php

```diff
@@ -114,19 +114,49 @@ public static function resolveAssociatedIdentity(
         if (is_null($currentIdentity->user_id)) {
             return null;
         }
-        $sql = 'SELECT
-                    i.*
-                FROM
-                    Identities i
-                INNER JOIN
-                    Emails e
-                ON
-                    e.user_id = i.user_id
-                WHERE
-                    i.user_id = ?
-                    AND (i.username = ? OR e.email = ?)
+        $sql = '(
+                    SELECT
+                        i.*
+                    FROM
+                        Identities i
+                    INNER JOIN
+                        Emails e
+                    ON
+                        e.user_id = i.user_id
+                    WHERE
+                        i.user_id = ?
+                        AND (i.username = ? OR e.email = ?)
+                )
+                UNION
+                (
+                    SELECT
+                        i.*
+                    FROM
+                        Identities i
+                    INNER JOIN
+                        Teams t
+                    ON
+                        i.identity_id = t.identity_id
+                    INNER JOIN
+                        Team_Users tu
+                    ON
+                        tu.team_id = t.team_id
+                    INNER JOIN
+                        Users u
+                    ON
+                        tu.user_id = u.user_id
```

Comment:
```suggestion
                        tu.user_id = u.user_id AND
                        i.user_id = u.user_id
```

qué pasa si se agrega esto?

Reply:
Mismo caso que los comentarios anteriores. `i.user_id` de las identidades para equipos tiene un valor `null` y nunca haría join con nada.